frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -127,12 +126,23 @@ action.GitMachete.BaseSlideInBranchBelowAction.notification.message.slide-in-und
 action.GitMachete.BaseSlideInBranchBelowAction.task-title=Sliding in
 
 
+action.GitMachete.BaseSlideOutBranchAction.deletion-suggestion-dialog.title=Git Machete Branch Deletion Suggestion
+action.GitMachete.BaseSlideOutBranchAction.deletion-suggestion-dialog.ok-text=&Delete Local Branch
```

Comment:
Hmm so it's sometimes `&` and sometimes `_` for the shortcut? 🤔

Reply:
yup. the `&` are for kotlin ui dsl button 

--- 

actually the `_` is an archaism. see the note there: https://jetbrains.org/intellij/sdk/docs/reference_guide/localization_guide.html